src/main.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+const form = document.querySelector('.form');
+
+form.addEventListener('submit', handleEvent);
+
+function handleEvent(event) {
+  event.preventDefault();
+  form.reset();
```

Comment:
Form reset isn't the best choice for form's submit, it'd better to remove this handler